pom.xml

```diff
@@ -9,17 +9,32 @@
 
     <properties>
         <java.version>1.8</java.version>
-        <handlebars-java-version>4.0.3</handlebars-java-version>
+        <handlebars-java-version>4.3.1</handlebars-java-version>
+        <handlebars-markdown-java-version>4.2.1</handlebars-markdown-java-version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <powermock.version>1.5.6</powermock.version>
-        <dp-logging.version>v2.0.0-beta.8</dp-logging.version>
-        <fasterxml.jackson.version>2.15.0</fasterxml.jackson.version>
+        <dp-logging.version>v2.0.0-beta.10</dp-logging.version>
+        <fasterxml.jackson.version>2.16.1</fasterxml.jackson.version>
         <jetty.version>9.4.53.v20231009</jetty.version>
-        <flying-saucer.version>9.1.22</flying-saucer.version>
+        <flying-saucer.version>9.4.0</flying-saucer.version>
     </properties>
 
     <dependencies>
-
+        <dependency>
```

Comment:
Same question: how come we need this?

Also, there are other logback exclusions in this file, with comments saying that AND Digital will deal with those, but they're still there. Is this something that needs checking as part of this ticket, or will that be done as part of a different ticket?

Reply:
I checked the exclusions and removed the ones that were not needed.